src/xdocs/config_naming.xml

```diff
@@ -2001,6 +2001,17 @@ class MyClass {
         </p>
         <source>
 &lt;module name=&quot;MethodName&quot;/&gt;
+        </source>
+        <p>Code Example:</p>
+        <source>
+class MyClass {
+  public void firstMethod1() {} // OK
+  protected void secondMethod2() {} // OK
+  private void ThirdMethod3() {} // violation, name 'ThirdMethod' must match
+                                 // pattern '^[a-z][a-zA-Z0-9]*$'
+  void FourthMethod4() {} // violation, name 'FourthMethod' must match
```

Comment:
This name is same as the above relative to the format. 
If you are adding one more example than add it with some different style, like you could use `_` or other similar mistakes with naming methods.
Also, add a public/protected/private modifier here.

Reply:
Resolved